scripts/ida_toggles.py

```diff
@@ -42,22 +43,98 @@ def add_toggle_data(self, dump_file_path):
                 raise
         self._add_toggle_data(dump_contents)
 
+    def _get_toggle_type(self, row_data):
+        """
+        Assign toggle type to model types
+        """
+
+        # convert sql dump model name to annotation report toggle type name
+        # this is usually the case with just devstack data dump, particularly from ecommerce
+        if row_data['model'] == "waffle.flag":
+            toggle_type = ToggleTypes.WAFFLE_FLAG
+        elif row_data['model'] == "waffle.switch":
+            toggle_type = ToggleTypes.WAFFLE_SWITCH
+        elif row_data['model'] == "waffle.sample":
+            toggle_type = ToggleTypes.WAFFLE_SAMPLE
+        else:
+            try:
+                toggle_type = ToggleTypes(row_data['model'])
+            except:
+                LOGGER.warning(
+                'Name of model not recognized: {}'.format(row_data['model'])
+                )
+                toggle_type = ToggleTypes.UNKNOWN
```

Comment:
Nit: You could move this to a ToggleTypes method like `get_toggle_type_from_table_name(table_name)`, and this becomes:
```
toggle_type = ToggleTypes.get_toggle_type_from_table_name(row_data['model'])
```